api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -1400,26 +1393,25 @@ public void test_updateAccountProperties_bypass_requests() {
     // user has no bypasses at test start
     assertThat(original.getDataUseAgreementBypassTime()).isNull();
     assertThat(original.getComplianceTrainingBypassTime()).isNull();
-    assertThat(original.getBetaAccessBypassTime()).isNull();
     assertThat(original.getEraCommonsBypassTime()).isNull();
     assertThat(original.getTwoFactorAuthBypassTime()).isNull();
 
     final List<AccessBypassRequest> bypasses1 =
         ImmutableList.of(
             new AccessBypassRequest().moduleName(AccessModule.DATA_USE_AGREEMENT).isBypassed(true),
-            new AccessBypassRequest().moduleName(AccessModule.COMPLIANCE_TRAINING).isBypassed(true),
             // would un-bypass if a bypass had existed
-            new AccessBypassRequest().moduleName(AccessModule.BETA_ACCESS).isBypassed(false));
```

Comment:
This test involves doing various things to arbitrary bypassable modules.

Here I'm substituting Compliance where Beta was used before.